src/NLog.Web.AspNetCore/LayoutRenderers/AspNetTlsHandshakeLayoutRenderer.cs

```diff
@@ -0,0 +1,63 @@
+﻿using NLog.LayoutRenderers;
+using NLog.Web.Internal;
+using System.Text;
+
+namespace NLog.Web.LayoutRenderers
+{
+    /// <summary>
+    /// ASP.NET TSL Handshake
+    /// </summary>
+    /// <remarks>
+    /// ${aspnet-tls-handshake}
+    /// </remarks>
+    [LayoutRenderer("aspnet-tls-handshake")]
+    public class AspNetTlsHandshakeLayoutRenderer : AspNetLayoutRendererBase
+    {
+        /// <summary>
+        /// Specifies which of the 7 properties of ITlsHandshakeFeature to emit
+        /// Defaults to the protocol
+        /// </summary>
+        public TlsHandshakeProperty Property { get; set; } = TlsHandshakeProperty.Protocol;
```

Comment:
Can you add the attribute: `[DefaultParameter]`

Reply:
Added as requested